src/Sulu/Bundle/AdminBundle/Resources/js/components/Button/Button.js

```diff
@@ -58,14 +57,20 @@ export default class Button extends React.PureComponent<Props> {
             skin,
             type,
         } = this.props;
+
+        const activeClass = {};
+        if (active && activeClassName) {
+            activeClass[activeClassName] = true;
+        }
+
         const buttonClass = classNames(
             buttonStyles.button,
             buttonStyles[skin],
             {
                 [buttonStyles.loading]: loading,
                 [buttonStyles.active]: active,
-                [activeClassName]: active,
             },
```

Comment:
Instead of the above if you could make use of the `classNames` functionality here:
```javascript
classNames({
    [activeClassName]: active && activeClassName,
});
```

Reply:
Flow tells me that i cannot assign computed property using undefined. This was the reason why i set the prop to required and the default value to an empty string the first time.

Reply:
Found another way to solve it.